charts/psmdb-operator/templates/deployment.yaml

```diff
@@ -56,6 +56,8 @@ spec:
               value: "{{ .Values.env.resyncPeriod }}"
             - name: LOG_VERBOSE
               value: "{{ .Values.env.logVerbose }}"
+            - name: DISABLE_TELEMETRY
+              value: {{ .Values.disableTelemetry }}
```

Comment:
Could you please make it the same as in PXC chart here: https://github.com/percona/percona-helm-charts/blob/main/charts/pxc-operator/templates/deployment.yaml#L61
(with quotes)

Reply:
done